go/vt/vttablet/tabletserver/query_executor_test.go

```diff
@@ -1242,6 +1242,31 @@ func TestQueryExecutorPlanPassSelect(t *testing.T) {
 	}
 }
 
+func TestQueryExecutorPlanSelectImpossible(t *testing.T) {
```

Comment:
As mentioned above, we'll need a test for the `select :bv from t where 1 != 1` case.